web-console/src/views/query-view/column-tree/column-tree.tsx

```diff
@@ -16,57 +16,286 @@
+): void {
+  if (columnSchema === 'druid') {
+    if (nodeData.icon === IconNames.TIME) {
+      onQueryStringChange(`SELECT
+  TIME_FLOOR(${escapeSqlIdentifier(String(nodeData.label))}, 'PT1H') AS "Time",
+  COUNT(*) AS "Count"
+FROM ${escapeSqlIdentifier(columnTable)}
+WHERE "__time" >= CURRENT_TIMESTAMP - INTERVAL '1' DAY
+GROUP BY 1
+ORDER BY "Time" ASC`);
+    } else {
+      onQueryStringChange(`SELECT
+  "${nodeData.label}",
+  COUNT(*) AS "Count"
+FROM ${escapeSqlIdentifier(columnTable)}
+WHERE "__time" >= CURRENT_TIMESTAMP - INTERVAL '1' DAY
+GROUP BY 1
+ORDER BY "Count" DESC`);
+    }
+  } else {
+    onQueryStringChange(`SELECT
+  ${escapeSqlIdentifier(String(nodeData.label))},
+  COUNT(*) AS "Count"
+FROM ${columnSchema}.${columnTable}
+GROUP BY 1
+ORDER BY "Count" DESC`);
+  }
+}
+
+function getColumnQuery(columnSchema: string, columnTable: string, nodeData: ITreeNode): string {
+  if (columnSchema === 'druid') {
+    if (nodeData.icon === IconNames.TIME) {
+      return `SELECT
+  TIME_FLOOR(${escapeSqlIdentifier(String(nodeData.label))}, 'PT1H') AS "Time",
+  COUNT(*) AS "Count"
+FROM ${escapeSqlIdentifier(columnTable)}
+WHERE "__time" >= CURRENT_TIMESTAMP - INTERVAL '1' DAY
+GROUP BY 1
+ORDER BY "Time" ASC`;
+    } else {
+      return `SELECT
+  "${nodeData.label}",
+  COUNT(*) AS "Count"
+FROM ${escapeSqlIdentifier(columnTable)}
+WHERE "__time" >= CURRENT_TIMESTAMP - INTERVAL '1' DAY
+GROUP BY 1
+ORDER BY "Count" DESC`;
+    }
+  } else {
+    return `SELECT
+  ${escapeSqlIdentifier(String(nodeData.label))},
+  COUNT(*) AS "Count"
+FROM ${columnSchema}.${columnTable}
+GROUP BY 1
+ORDER BY "Count" DESC`;
+  }
+}
+
 export interface ColumnTreeProps {
   columnMetadataLoading: boolean;
   columnMetadata?: ColumnMetadata[];
   onQueryStringChange: (queryString: string) => void;
+  defaultSchema?: string;
+  defaultTable?: string;
 }
 
 export interface ColumnTreeState {
   prevColumnMetadata?: ColumnMetadata[];
   columnTree?: ITreeNode[];
   selectedTreeIndex: number;
+  expandedNode: number;
 }
 
 export class ColumnTree extends React.PureComponent<ColumnTreeProps, ColumnTreeState> {
   static getDerivedStateFromProps(props: ColumnTreeProps, state: ColumnTreeState) {
-    const { columnMetadata } = props;
+    const { columnMetadata, defaultSchema, defaultTable } = props;
 
     if (columnMetadata && columnMetadata !== state.prevColumnMetadata) {
+      const columnTree = groupBy(
+        columnMetadata,
+        r => r.TABLE_SCHEMA,
+        (metadata, schema): ITreeNode => ({
+          id: schema,
+          label: schema,
+          childNodes: groupBy(
+            metadata,
+            r => r.TABLE_NAME,
+            (metadata, table) => ({
+              id: table,
+              icon: IconNames.TH,
+              label: (
+                <Popover
+                  boundary={'window'}
+                  position={Position.RIGHT}
+                  content={basicActionsToMenu([
```

Comment:
No need to use `basicActionsToMenu` I want to get rid of this function completely, maybe factor out the menus into functions